src/mz390.java

```diff
@@ -993,7 +995,7 @@ public  class  mz390 {
 			2, 2, 3, 2, 0, 2, 2, 8, 2, 3, 2, 2, 2, 2, 3, // 2 * / prev mpy/div  RPI 214
 			3, 3, 3, 4, 3, 0, 3, 3, 3, 3, 3, 3, 3, 3, 3, // 3 (   prev open (...) RPI 274, RPI 647
 			3, 3, 3,11, 3, 3, 3, 8,11, 3, 3, 3, 3, 3, 3, // 4 )   prev var subscript RPI 559
-			0, 0, 3, 5, 5, 5, 5, 8, 0, 3, 0, 0, 0, 0, 3, // 5 .   prev concat
+			5, 5, 3, 5, 5, 5, 5, 8, 0, 3, 0, 0, 0, 0, 3, // 5 .   prev concat
```

Comment:
I would like to see a test program that I can assemble that has the problem without the fix and is okay with the fix.

Reply:
I created a short test program (see attached files) that has a test macro that uses the code snippet provided for the INDEX issue. The above new line fixed the problem.

Unrelated to the current pull request, the test program uncovered another issue. If you look at the test output, the first of the 3 tests yields the "invalid" MNOTE, as expected. However, the value for the &D SETA variable is -1, as expected, but is displayed as 1 (see the added MNOTEs in the output). I will open a new issue for this.

[TINDEX.MLC.TXT](https://github.com/z390development/z390/files/9641149/TINDEX.MLC.TXT)
[TINDEX.PRN.TXT](https://github.com/z390development/z390/files/9641150/TINDEX.PRN.TXT)

Reply:
I just tried the negative test on my TK4 (MVS) system. It also displayed the negative value as a positive value. Since this system uses ASMF, it would be nice to see what HLASM displays. Right now I will hold off opening a new issue.